Formula/phylum.rb

```diff
@@ -0,0 +1,24 @@
+class Phylum < Formula
+  desc "Software Supply Chain Security CLI"
+  homepage "https://phylum.io"
+  url "https://github.com/phylum-dev/cli/archive/refs/tags/v3.12.1.tar.gz"
+  sha256 "631ecb17a8100facd0b9e82e597c737cd105c0734408b9f68dec1aea99205e57"
+  license "GPL-3.0-or-later"
+  head "https://github.com/phylum-dev/cli.git", branch: "main"
+
+  bottle do
+    root_url "https://github.com/phylum-dev/homebrew-cli/releases/download/phylum-3.12.1"
+    sha256 cellar: :any_skip_relocation, monterey:     "766cff9de998a467d6f6174a27e01c0ce0b73f2d459ad6e14f578f2393df3428"
+    sha256                               x86_64_linux: "ca45cb0c23f551ae3e55e942c2716d4fba6526ebf51720649d130d9b68c49aed"
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--no-default-features", *std_cargo_args(path: "cli")
```

Comment:
This is the standard way to install rust packages with brew. You can see this in any of the formulae for rust projects in [homebrew-core](https://github.com/Homebrew/homebrew-core/tree/master/Formula) and it is generated by [the formula creator](https://github.com/Homebrew/brew/blob/72f7880e2a2d5bd2e1c53b2a425e40b92c50c95c/Library/Homebrew/formula_creator.rb#L207-L208) when you run `brew create --rust <URL>`.

Homebrew controls `$PATH` during package build and installation and will ensure that `cargo` is available because of the `depends_on "rust"` line. Additionally, the [`std_cargo_args`](https://github.com/Homebrew/brew/blob/72f7880e2a2d5bd2e1c53b2a425e40b92c50c95c/Library/Homebrew/formula.rb#L1521-L1525) includes a `--root` to ensure that the package is installed into the correct location.

For some additional context, Homebrew installs each package to its own "Cellar" and then uses symlinks to expose binaries on `$PATH`:

```
❯ which phylum; realpath $(which phylum)
/opt/homebrew/bin/phylum
/opt/homebrew/Cellar/phylum/3.12.1/bin/phylum
```

The "Cellar" (in this case `/opt/homebrew/Cellar/phylum/3.12.1`) can then be put into a "Bottle" for a binary distribution.

Reply:
Fair enough, I guess I was asking too much of brew.